features/admin.applications.v1/components/edit-application.tsx

```diff
@@ -875,7 +876,7 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
             }
             if (isFeatureEnabled(featureConfig?.applications,
                 ApplicationManagementConstants.FEATURE_DICTIONARY.get("APPLICATION_EDIT_ADVANCED_SETTINGS"))
-                && !isFragmentApp
+                && !isSubOrganization()
```

Comment:
Shouldn't `isFragmentApp` be already true in sub organizations?

Reply:
Since from [1] we are allowing to create applications in sub organizations, we need to hide this from this level.

[1] https://github.com/wso2/product-is/issues/21208